deno_typescript/ops.rs

```diff
@@ -41,21 +41,27 @@ struct ReadFile {
   should_create_new_source_file: bool,
 }
 
-pub fn read_file(s: &mut TSState, v: Value) -> Result<Value, ErrBox> {
+pub fn read_file(_s: &mut TSState, v: Value) -> Result<Value, ErrBox> {
   let v: ReadFile = serde_json::from_value(v)?;
   let (module_name, source_code) = if v.file_name.starts_with("$asset$/") {
     let asset = v.file_name.replace("$asset$/", "");
 
-    let source_code = match s.custom_assets.get(&asset) {
-      Some(asset_path) => std::fs::read_to_string(&asset_path)?,
-      None => crate::get_asset2(&asset)?.to_string(),
+    let source_code = match crate::get_asset(&asset) {
+      Some(code) => code.to_string(),
+      None => {
+        return Err(
+          std::io::Error::new(std::io::ErrorKind::NotFound, "Asset not found")
+            .into(),
+        );
+      }
     };
 
     (asset, source_code)
   } else {
     assert!(!v.file_name.starts_with("$assets$"), "you meant $asset$");
     let module_specifier = ModuleSpecifier::resolve_url_or_path(&v.file_name)?;
     let path = module_specifier.as_url().to_file_path().unwrap();
+    // TODO(bartlomieju): has no effect on cli crate, still rebuilds every time
     println!("cargo:rerun-if-changed={}", path.display());
```

Comment:
I'm not sure this comment is true. If you "touch cli/lib.rs" it will not cause a rebuild of the snapshot, but if you do "touch cli/js/main.ts" it will rebuild the snapshot - which is what we want.

Reply:
Oooops, forgot to remove.